custom_components/acwd/helpers.py

```
@@ -30,14 +31,14 @@ def parse_api_response(result: dict, endpoint: str = "unknown") -> Any:
     Raises:
         ValueError: If 'd' is absent or its value is not valid JSON.
     """
-    if "d" not in result:
+    if KEY_D not in result:
         raise ValueError(
-            f"Unexpected API response from {endpoint}: missing 'd' property"
+            f"Unexpected API response from {endpoint}: missing '{KEY_D}' property"
         )
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)